design-system-docs/src/_component_docs/_defaults.yml

```diff
@@ -0,0 +1 @@
+show_on_this_page: true
```

Comment:
This _defaults.yml file is super useful, it lets you set common front-matter for any files within this directory.